### PR TITLE
Markdown linting

### PR DESCRIPTION
**Pull request by sudo-bmitch**
Signed-off-by: Brandon Mitchell <git@bmitch.net>

This is cleaning up a bunch of linting warnings that were bugging me:

- linefeed between heading and sections
- using `-` for all unordered list for consistency
- using `1. ` for all ordered list entries to allow easier insertion to the middle of a list
- one sentence per line for easier PRs
- headings without a trailing `:` or other punctuation
- alt-text for images
- specifying syntax for each code block
- removing trailing whitespace
- probably something else I'm forgetting

### CODE REVIEW

File: keymanagementrequirements.md [hunk 1]
```diff
@@ -1,79 +1,124 @@
...
-    - Publisher Signer: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Signers (i.e. developers, signing hosts) will be responsible for signing artifacts.
-- Deployer: User who deploys containers.
-    - Deployer Admin: In some scenarios a deployer will include a group of users (i.e. teams or enterprises). Admin users (i.e. security administrators) will be responsible for configuring deployment policies, including potentially a set of trusted roots or required signatures. 
-    - Deployer Operator: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Operators (i.e. developers, deployment hosts) will be responsible for pulling container images from registries, verifying their signatures, and then running them.
-- Repository Owner: In some scenarios a container may be stored in a repository that is managed by the repository owner. The repository may be public or private and could also be air-gapped.
+  - Publisher Admin:
```
Comment (SteveLasker) [inline keymanagementrequirements.md:13]: @sudo-bmitch, were you looking for a linewrap on the output? 
- Publisher Admin:  
  In some scenarios a publisher will include a group of users (i.e. teams or enterprises).
    Admin users (i.e. security administrator) will be responsible for configuring roots, provide access to use or generate delegate keys, and make decisions for key revocation.

**Review by SteveLasker — COMMENTED** [review]
General LGTM, with 1 clarifying question on expected formatting

File: keymanagementrequirements.md (hunk 1 above)
Comment (SteveLasker) [reply keymanagementrequirements.md:13]: ping: @sudo-bmitch

Comment (sudo-bmitch) [reply keymanagementrequirements.md:13]: Woops, sorry @SteveLasker , missed that previous notification. I'm seeing the lines wrap in my view. Are they getting split in yours? In general I was keeping each sentence to a separate line to make PRs easier.

**Review by SteveLasker — APPROVED** [review]
LGTM

**Comment by SteveLasker** [comment]
Back to @sudo-bmitch for merge conflicts

**Pull request merged**